src/org/zaproxy/zap/extension/pscanrules/CrossDomainScriptInclusionScanner.java

```diff
@@ -116,7 +119,18 @@ private boolean isScriptFromOtherDomain (String host, String scriptURL){
 			String scriptHost = scriptURI.getHost();
 			if(scriptHost != null && !scriptHost.toLowerCase().equals(host.toLowerCase())){
 				result = true;
-			} 
+			}
+			if(Plugin.AlertThreshold.HIGH.equals(this.getLevel())) {

```

Comment:
This if statement could also use the variable `result` (i.e. no need to check further if it's not a script from other domain).
(The variable could be renamed to `otherDomain`.)